### PR TITLE
Corrected Grammatical errors

### PR DESCRIPTION
**Pull request by JunaidPeer**

### CODE REVIEW

**Comment by alamshafil** [comment]
Thank you for your work! I changed the branch from master to 4.1-dev to prevent any merge conflicts in the future. If you could, please resolve the conflicts for 4.1. If you are confused, tell me and I can try to fix the conflicts from my end.

File: README.md [hunk 1]
```diff
@@ -125,16 +125,13 @@ make -j8
 
 # About Avian
 
-A digital peer to peer network for the facilitation of asset transfer.
+A digital peer-to-peer network for the facilitation of asset transfer.
```
Comment (alamshafil) [inline README.md:128]: 4.1 has this:
```Avian digital peer to peer network for the facilitation of asset transfer.```

**Review by alamshafil — CHANGES_REQUESTED** [review]
Fix merge conflicts

File: README.md (hunk 1 above)
Comment (cdonnachie) [reply README.md]: I have corrected the conficts.

**Review by alamshafil — APPROVED** [review]

**Comment by alamshafil** [comment]
ACK

**Pull request merged**